Models/Instruments/ND/canvas/ND.nas

```diff
@@ -417,14 +418,14 @@ for (i = 0; i < 2; i = i + 1 ) {
 	});
 }
 
-setlistener("/instrumentation/efis[0]/nd/terrain-on-nd", func{
-	var terr_on_hd = getprop("instrumentation/efis[0]/nd/terrain-on-nd");
-	var alpha = 1;
-	if (terr_on_hd) {
-		alpha = 0.5;
-	}
-	nd_display.main.setColorBackground(0,0,0,alpha);
-});
+#setlistener("/instrumentation/efis[0]/nd/terrain-on-nd", func{
```

Comment:
What are these commented out changes, are they still WIP?

Reply:
It seems useless on my implementation. 
What it does?

Reply:
OK,then it can be removed entirely?